README.md

```diff
@@ -1515,6 +1515,265 @@ __NOTE__: This section does not go over any additional tuning that can be done
+# LIO_MGMT_IP4_MASK=16
+
+# modprobe bonding
+# ip link set $PF0 up
+# ip link set $PF1 up
+# ip link add $LIO_MACVLAN_PF0 link $PF0 type macvlan
+# ip link add $LIO_MACVLAN_PF1 link $PF1 type macvlan
+
+# ip link add $LIO_BOND_MGMT type bond
+# echo balance-rr > /sys/class/net/$LIO_BOND_MGMT/bonding/mode
+# ip link set $LIO_MACVLAN_PF0 master $LIO_BOND_MGMT
+# ip link set $LIO_MACVLAN_PF1 master $LIO_BOND_MGMT
+
+# ip addr add $LIO_HOST_MGMT_IP4_ADDR/$LIO_MGMT_IP4_MASK dev $LIO_BOND_MGMT
+# ip link set $LIO_BOND_MGMT up
+```
+
+A quick test with ping after the above steps should confirm that DUT can reach the network stack within the adapter.
+
+```
+# ping -q 169.254.1.1 -c 1
+
+PING 169.254.1.1 (169.254.1.1) 56(84) bytes of data.
+
+--- 169.254.1.1 ping statistics ---
+1 packets transmitted, 1 received, 0% packet loss, time 0ms
+rtt min/avg/max/mdev = 0.101/0.101/0.101/0.000 ms
+```
+
+### Create dummy ovs-* binaries
+
+Using the lio-bond-mgmt interface created above, we will redirect all OVS commands from remote tester machine
+to OVS control plane running on NIC.
+
+Create a directory in DUT under /tmp/ to hold our dummy binaries:
+
+```
+# mkdir -p /tmp/ovs-bin/
+```
+
+Create ovs-vswitchd file in /tmp/ovs-bin and write below commands to it so that all
+ovs commands will be sent to the OVS control plane on the NIC via ssh.
+
+```
+#!/bin/bash
+sshpass -p <NIC-PASSWORD> ssh root@169.254.1.1 "sh -l -c \"$(basename $0) $@\""
+
+Where, NIC-PASSWORD is the root password to access the NIC control plane. Please check with Cavium support to obtain the 
+password for your NIC.
+```
+
+Create a link for all ovs-* binaries to the new ovs-vswitchd:
+
+```
+# cd /tmp/ovs-bin
+# ln -sf ovs-vswitchd ovs-vsctl
+# ln -sf ovs-vswitchd ovs-ofctl
+# ln -sf ovs-vswitchd ovs-appctl
+# ln -sf ovs-vswitchd ovs-dpctl
+
+# chmod +x ovs-vswitchd
+```
+
+Update ~/.bashrc to include /tmp/ovs-bin in its path
+
+```
+# echo "export PATH=/tmp/ovs-bin/:\$PATH" >> ~/.bashrc
+# source ~/.bashrc
+```
+
+### Finding Physical and Virtual interfaces for test
+
+OVS control plane running on the NIC has a virtual interface corresponding to each Physical
+and Virtual function on DUT. These virtual interfaces are used for all OVS operations within the NIC.
+
+Following is the naming convention followed:
+
+__Physial Functions__: Each physical interface is named as `ethN`, N=0/1, depending upon the physical port being used.  
+  For this example, we'll pass `--physical-interface eth0` as parameter to ovs_performance.py.
+
+__Virtual Functions__: Each  virtual interface is named as `enp`$BUS`s`$DEV`f`$FUNC`, where BUS,DEV and FUNC are PCI Bus, Device and Function
+  for that interface.
+
+To get the network interface belonging to a specific virtual PCI function:
+
+Get the PCI ID for VF:
+
+```
+# lspci -d 177d:9712
+05:00.3 Ethernet controller: Cavium, Inc. CN23XX [LiquidIO II] SRIOV Virtual Function (rev 02)
+
+This command showss the BUS-DEV-FUNC for the VF device as 5-0-3
```

Comment:
This command shows the BUS-DEV-FUNC for the VF device as 5-0-3:

showss = shows

Please